app-modules/ai/database/migrations/2024_05_14_212612_create_ai_assistant_upvotes_table.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('ai_assistant_upvotes', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->foreignUuid('assistant_id')->constrained()->cascadeOnDelete();
+            $table->foreignUuid('user_id')->constrained()->cascadeOnDelete();
```

Comment:
We should confirm with Product as to whether or not this should be deleted or nulled if the user is deleted.

We may want to persist upvotes even if a User is removed.

Reply:
I was just copying the other upvote features like prompt upvotes here